.github/workflows/test-action.yml

```diff
@@ -39,6 +39,11 @@ jobs:
       - name: Setup dependencies
         run: make docker-deps
 
+      - name: Reset master
```

Comment:
why is that needed?

Reply:
Right now credo only warns for the new errors added, not for the existing ones. To be able to do that it compares the errors with the errors in master. Github does not fetch master by default, so this is why we have to create it before credo runs